tests/components/gearvr-controls.test.js

```diff
@@ -149,8 +151,8 @@ suite('gearvr-controls', function () {
       component.checkIfControllerPresent();
 
       // Fail purposely.
-      el.addEventListener('thumbstickmoved', function (evt) {
-        assert.ok(false);
+      el.addEventListener('trackpadmoved', function (evt) {
```

Comment:
This was previously incorrect. Likely since it was copied from the oculus-touch-controls.test.js I think. Once I changed it I was able to verify that I could indeed hit the fail when I updated the changed value accordingly.

Reply:
good catch 👍